src/content/en/fundamentals/media/manipulating/live-effects.md

```diff
@@ -0,0 +1,374 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Many of today's most popular apps let you apply filters and effects to images or video. This article talks about how to implement these features on the open web.
+
+{# wf_updated_on: 2017-08-08 #}
+{# wf_published_on: 2017-08-08 #}
+{# wf_blink_components: Blink>WebGL, Blink>Canvas #}
+
+# Live effects for images and video {: .page-title }
```

Comment:
I don’t like the title that much, honestly. I think it’s the work “live”. Maybe rather “real-time”?